src/features/cartographie/infrastructure/presentation/layouts/cartographie/cartographie.layout.html

```diff
@@ -1,7 +1,26 @@
 <!--<app-structure-list-search></app-structure-list-search>-->
 <div class="d-flex h-100">
-  <div class="col-xxl-3 col-xl-4 col-lg-4 col-md-6 col-12 overflow-auto">
+  <div class="col-xxl-5 col-xl-5 col-lg-5 col-md-6 col-12 overflow-auto">
```

Comment:
Tester si le omportement est le même avec :
```suggestion
  <div class="col-lg-5 col-md-6 col-12 overflow-auto">
```